plugins/ui/src/js/package.json

```
@@ -42,21 +42,21 @@
   },
   "dependencies": {
     "@adobe/react-spectrum": "^3.34.1",
-    "@deephaven/chart": "^0.78.0",
-    "@deephaven/components": "^0.78.0",
-    "@deephaven/dashboard": "^0.78.0",
-    "@deephaven/dashboard-core-plugins": "^0.78.0",
-    "@deephaven/grid": "^0.78.0",
-    "@deephaven/icons": "^0.78.0",
-    "@deephaven/iris-grid": "^0.78.0",
-    "@deephaven/jsapi-bootstrap": "^0.78.0",
-    "@deephaven/jsapi-components": "^0.78.0",
-    "@deephaven/jsapi-types": "^1.0.0-dev0.34.2",
-    "@deephaven/log": "^0.78.0",
-    "@deephaven/plugin": "^0.78.0",
-    "@deephaven/react-hooks": "^0.78.0",
-    "@deephaven/redux": "^0.78.0",
-    "@deephaven/utils": "^0.78.0",
+    "@deephaven/chart": "^0.81.1",
+    "@deephaven/components": "^0.81.1",
+    "@deephaven/dashboard": "^0.81.1",
+    "@deephaven/dashboard-core-plugins": "^0.81.1",
+    "@deephaven/grid": "^0.81.0",
+    "@deephaven/icons": "^0.81.0",
+    "@deephaven/iris-grid": "^0.81.1",
+    "@deephaven/jsapi-bootstrap": "^0.81.1",
+    "@deephaven/jsapi-components": "^0.81.1",
+    "@deephaven/jsapi-types": "^1.0.0-dev0.33.4",
```

Should still be 0.34.2, which is actually the "latest" version:
```suggestion
    "@deephaven/jsapi-types": "^1.0.0-dev0.34.2",
```

`v0.33.4` was incorrectly tagged as `latest`, and I've filed a ticket to resolve that: https://github.com/deephaven/deephaven-core/issues/5572

---

fixed